crmprtd/align.py

```diff
@@ -58,6 +58,12 @@ def convert_unit(val, src_unit, dst_unit):
                              'dst_unit': dst_unit,
                              'exception': e})
             return None
+        except DimensionalityError as e:
```

Comment:
Since the handling of the exception is the same for both UndefinedUniterror and DimensionalityError, you should test for both of those together:

```
try:
    ...
except (UndefinedUnitError, DimensionalityError) as e:
    ...
```

Reply:
Ok cool, I didn't know you were able to do that!